patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+-            return SensorNearestLivingEntities.a.a(entityliving, entityliving1);
+-        }).collect(Collectors.toList()));
++        List<EntityLiving> result = new ArrayList<>();
++        for (EntityLiving entityliving1 : list) {
++            if (SensorNearestLivingEntities.a.a(entityliving, entityliving1)) {
++                result.add(entityliving1);
++            }
++        }
++        behaviorcontroller.setMemory(MemoryModuleType.VISIBLE_MOBS, result);
+     }
+ 
+     @Override
+diff --git a/src/main/java/net/minecraft/server/TagRegistry.java b/src/main/java/net/minecraft/server/TagRegistry.java
+index c836cd8f7945d9209c71ca7f1afebb43b931da18..f212315a694325714096d18d56d4737975c386d6 100644
+--- a/src/main/java/net/minecraft/server/TagRegistry.java
++++ b/src/main/java/net/minecraft/server/TagRegistry.java
+@@ -2,7 +2,11 @@ package net.minecraft.server;
+ 
+ import com.google.common.collect.HashMultimap;
+ import com.google.common.collect.Multimap;
++
++import java.util.ArrayList;
++import java.util.List;
+ import java.util.Map;
++import java.util.StringJoiner;
+ import java.util.concurrent.CompletableFuture;
+ import java.util.concurrent.Executor;
+ import java.util.stream.Collectors;
+@@ -82,9 +86,17 @@ public class TagRegistry implements IReloadListener {
+             multimap.putAll("fluids", TagsFluid.b(this.fluidTags));
+             multimap.putAll("entity_types", TagsEntity.b(this.entityTags));
+             if (!multimap.isEmpty()) {
+-                throw new IllegalStateException("Missing required tags: " + (String) multimap.entries().stream().map((entry) -> {
+-                    return (String) entry.getKey() + ":" + entry.getValue();
+-                }).sorted().collect(Collectors.joining(",")));
++                List<String> toSort = new ArrayList<>();
++                for (Map.Entry<String, MinecraftKey> entry : multimap.entries()) {
++                    String s = entry.getKey() + ":" + entry.getValue();
++                    toSort.add(s);
++                }
++                toSort.sort(null);
++                StringJoiner joiner = new StringJoiner(",");
++                for (String s : toSort) {
++                    joiner.add(s);
++                }
++                throw new IllegalStateException("Missing required tags: " + (String) joiner.toString());
+             }
+         }, executor1);
+     }
+diff --git a/src/main/java/net/minecraft/server/TickListChunk.java b/src/main/java/net/minecraft/server/TickListChunk.java
+index 8af1229c3da63a838b0bec1cafde1e41cf5cf551..7ee8534bc656e117fa2a21a39693d53ac8a776b8 100644
+--- a/src/main/java/net/minecraft/server/TickListChunk.java
++++ b/src/main/java/net/minecraft/server/TickListChunk.java
+@@ -84,9 +84,9 @@ public class TickListChunk<T> implements TickList<T> {
+     }
+ 
+     public void a(TickList<T> ticklist) {
+-        this.a.forEach((ticklistchunk_a) -> {
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/ArrayList.java#l1226